pkg/omada/handler.go

```diff
@@ -1,126 +1,126 @@
-package omada

```

Comment:
apologise for the code formatting

Reply:
Could you please reformat this file with a single tab indentation just so it matches the rest of the project? Happy to do this for you if you need